product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,213 @@
+---------------  -----------  ---------------  -------  --------------
+virtualMachines  westus2      Standard_DS2_v2  1,2,3    None
+```
+
+#### Check the resource limits for vCPUs and Public IP Addresses for your region
+
+To check if you have adequate Azure resources to provision new clusters:
+
+1. In the [Azure Portal](https://portal.azure.com/), select **Subscription**.
+2. Select your specific subscription.
+3. Select **Usage + quotas** in the **Settings** section.
+4. Search for *Total Regional vCPUs* and select the **Location** to check the regional vCPUs limits.
+5. Search for *Dv4* and *Esv3* to view virtual machine limits.
+6. Search for Public IP addresses to view network limits.
+
+## Step 2: Configure your Azure subscription
+
+After checking whether the resource requirements are met, configure your Azure subscription based on the output of the `azure-preflight.sh`. 
+
+### Register Azure Service Providers 
+To register Service Providers using the Azure Portal:
+
+1. In the [Azure Portal](https://portal.azure.com/), select **Subscription**.
+2. Select your specific subscription.
+3. In the navigation panel **Settings** group, select **Resource providers**.
+4. Review the status of the required providers. To register a provider, select the provider, and, on the top menu, select **Register**.
+
+To register Service Providers using the Azure CLI,
+use the [register command](https://docs.microsoft.com/en-us/cli/azure/provider?view=azure-cli-latest#az_provider_register). For example:
+
+```sh
+ az provider register -n Microsoft.ContainerService
+Registering is still on-going. You can monitor using 'az provider show -n Microsoft.ContainerService
+
+```
+
+### Fix issues with SKU restrictions
+
+[Open a support request](https://docs.microsoft.com/en-us/troubleshoot/azure/general/region-access-request-process)  to remove SKU restrictions in the designated region.
+
+### Increase Public IP Addresses limits
+
+Increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your clusters with the total number of clusters you plan to use.
+
+You can increase the number of public IP addresses for your account either by using Azure's portal if you have appropriate privileges or by submitting a support request. See:
+
+* [Request Networking quota increase at subscription level using Help + support](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/networking-quota-requests#request-networking-quota-increase-at-subscription-level-using-help--support)
+
+* [Request networking quota increase at subscription level using Usages + quotas](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/networking-quota-requests#request-networking-quota-increase-at-subscription-level-using-usages--quotas)
+
+
+### Increase vCPU limits
+
+You can increase the number of Esv3 family virtual machines per region either by: 
```

Comment:
You can increase the number of Dv4 or Esv3 family virtual machines per region either by:

Reply:
Added Dv4.